### ISSUE TITLE
XFA can't access property "find", fontFinder is undefined

### ISSUE THREAD

**Issue by JDziurlaj** [mozilla/pdf.js#13647]
Attach (recommended) or Link to PDF file here:
[E-Ticket.pdf](https://github.com/mozilla/pdf.js/files/6736099/E-Ticket.pdf)
Configuration:

- Web browser and its version: 91.0a1 (2021-06-29) (64-bit)
- Operating system and its version: Windows 10

Steps to reproduce the problem:
1. enableXfa = true
2. Open form

What is the expected behavior? (add screenshot)
PDF renders.
![image](https://user-images.githubusercontent.com/7104454/123860038-65b5d500-d8f3-11eb-8d1c-865f7de0ccd5.png)

What went wrong? (add screenshot)
Console error:
`Warning: XFA - an error occured during layout: TypeError: can't access property "find", fontFinder is undefined`

**Comment by marco-c** [comment]
This is now working. With mozregression, I found it was fixed in Firefox by the update in https://bugzilla.mozilla.org/show_bug.cgi?id=1727189.

**Comment by Snuffleupagus** [comment]
> This is now working.

Not really, given that this is a XFA Foreground document and those were (temporarily) disabled in PR #13660 since we don't support them yet. Hence this will be broken if/when we implement support for XFAF documents, so I'm re-opening this issue :-)

---

A simple way to tell, indirectly, for now if a document is a XFAF one is by checking the web console. If there's an `Warning: XFA support is not enabled` line printed *despite* the preference being set to `enableXFA = true` then it's an XFAF document.

**Comment by marco-c** [comment]
Ahah, you're right, I hadn't noticed that I can't edit the form!
